rust/osm2lanes/src/overpass.rs

```diff
@@ -138,35 +155,52 @@ pub async fn get_way(id: &ElementId) -> Result<(Tags, Locale), reqwest::Error> {
 ///
 pub async fn get_nearby(
     lng_lat: (f64, f64),
-) -> Result<(Tags, Locale, Vec<LatLon>), reqwest::Error> {
+) -> Result<(ElementId, Tags, Vec<LatLon>, Locale), reqwest::Error> {
     const RADIUS: f64 = 10.0_f64;
     let mut resp = reqwest::Client::new()
         .get(format!(
             r#"https://overpass-api.de/api/interpreter?data=[out:json][timeout:25];
             way
-            (around:{RADIUS},{},{})
-            ["highway"];
-            out tags geom;"#,
+                (around:{RADIUS},{},{})
```

Comment:
Nice combined query! The overpass docs honestly confuse me a fair bit; I just trial-and-error combine from bits of example

Reply:
I just assume everyone but the creator of overpass does that :P